org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/text/correction/SourceAssistProcessor.java

```diff
@@ -618,6 +638,64 @@ private Optional<Either<Command, CodeAction>> getFinalModifierWherePossibleActio
 		}
 	}
 
+	private Optional<Either<Command, CodeAction>> getSortMembersProposal(IInvocationContext context, CodeActionParams params, String kind, String label) {
+		CategorizedTextEditGroup group = new CategorizedTextEditGroup(label, new GroupCategorySet(new GroupCategory(label, label, label)));
+		try {
+			TextEdit edit = CompilationUnitSorter.sort(context.getASTRoot(), new DefaultJavaElementComparator(false), 0, group, null);
+			if (edit == null) {
+				return Optional.empty();
+			}
+			CodeActionProposal sortMembersProposal = (pm) -> {
+				return convertToWorkspaceEdit(context.getCompilationUnit(), edit);
+			};
+			return getCodeActionFromProposal(params.getContext(), context.getCompilationUnit(), label, kind, sortMembersProposal, CodeActionComparator.SORT_MEMBERS_PRIORITY);
+		} catch (JavaModelException e) {
+			return Optional.empty();
+		}
+	}
+
+	private Optional<Either<Command, CodeAction>> getSortMembersForSelectionProposal(IInvocationContext context, CodeActionParams params, List<ASTNode> coveredNodes) {
+		String label = ActionMessages.SortMembers_selectionLabel;

```

Comment:
Looks like `label` isn't set to another value anywhere here. Could we just use `ActionMessages.SortMembers_selectionLabel` where needed ?

Reply:
Sure.